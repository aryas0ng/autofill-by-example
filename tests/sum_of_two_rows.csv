11,26
15,17
2,7
5,8
3,
2,
4,
43,
5,
6,
9,
3,
4,
2,
5,
2,
6,
1,
8,
6,
1,
21,
3,