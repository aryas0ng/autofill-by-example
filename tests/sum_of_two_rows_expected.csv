11,26
15,17
2,7
5,8
3,5
2,6
4,47
43,48
5,11
6,15
9,12
3,7
4,6
2,7
5,7
2,8
6,7
1,9
8,14
6,7
1,22
21,24
3,3